filebeat/tests/integration/filestream_test.go

```diff
@@ -678,6 +678,50 @@ func TestFilestreamTakeOverFromLogInput(t *testing.T) {
 	}
 }
 
+func TestFilestreamHasOwnerAndGroup(t *testing.T) {
```

Comment:
Another important thing, the tests on this file also run on Windows, but this feature is not supported on Windows.

My recommendation is to put it on a separate file and add the `!windows` build flag:
```go
//go:build integration && !windows
```